frontend/src/components/rightbar/Rightbar.css

```diff
@@ -108,3 +108,16 @@
     animation:slidebg 2s linear infinite;
 }
 
+@media (max-width: 480px) {
```

Comment:
ich würde die Breite in rem angeben , nicht Pixel, sonst kann es auf Handy/Tablet/Laptop unterschiedlich ausschauen.